### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by WiNloSt**
### Description

Fix missing translations reported in this Slack thread
- https://metaboat.slack.com/archives/C864UT5CZ/p1691574707654909?thread_ts=1691422223.225969&cid=C864UT5CZ
- https://metaboat.slack.com/archives/C864UT5CZ/p1691577450690729?thread_ts=1691422223.225969&cid=C864UT5CZ
- https://metaboat.slack.com/archives/C864UT5CZ/p1691578019499279?thread_ts=1691422223.225969&cid=C864UT5CZ

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ No test has been added since this is just translating existing strings in the codebase.

### CODE REVIEW

**Review by oisincoveney — APPROVED** [review]

File: enterprise/frontend/src/metabase-enterprise/application_permissions/selectors.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const getApplicationPermissionEditor = createSelector(
 
       return {
         id: group.id,
-        name: group.name,
+        name: getGroupNameLocalized(group),
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/application_permissions/selectors.ts:95]: image

File: enterprise/frontend/src/metabase-enterprise/sandboxes/index.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ const getEditSegmentedAccessPostAction = (entityId, groupId, view) =>
 if (hasPremiumFeature("sandboxes")) {
   PLUGIN_ADMIN_USER_FORM_FIELDS.push({
     name: "login_attributes",
-    title: "Attributes",
+    title: t`Attributes`,
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/sandboxes/index.js:58]: image

File: enterprise/frontend/src/metabase-enterprise/sandboxes/components/LoginAttributesWidget.jsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
 /* eslint-disable react/prop-types */
+import { t } from "ttag";
 import MappingEditor from "./MappingEditor";
 
 const LoginAttributesWidget = ({ field }) => (
   <MappingEditor
     value={field.value || {}}
     onChange={field.onChange}
-    addText="Add an attribute"
+    addText={t`Add an attribute`}
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/sandboxes/components/LoginAttributesWidget.jsx:9]: image

File: frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx [hunk 4]
```diff
@@ -27,7 +27,7 @@ export const DataPermissionsHelp = () => {
         defaultValue="database-level"
       >
         <Accordion.Item value="database-level">
-          <Accordion.Control>Database levels</Accordion.Control>
+          <Accordion.Control>{t`Database levels`}</Accordion.Control>
```
Comment (WiNloSt) [inline frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx:30]: image

File: frontend/src/metabase/home/components/CustomHomePageModal/CustomHomePageModal.tsx [hunk 5]
```diff
@@ -89,19 +89,19 @@ export const CustomHomePageModal = ({
   return (
     <Modal isOpen={isOpen} onClose={onClose}>
       <ModalContent
-        title="Customize Homepage"
+        title={t`Customize Homepage`}
```
Comment (WiNloSt) [inline frontend/src/metabase/home/components/CustomHomePageModal/CustomHomePageModal.tsx:92]: image

File: frontend/src/metabase/home/components/HomeLayout/HomeLayout.tsx [hunk 6]
```diff
@@ -31,13 +32,13 @@ export const HomeLayout = ({
       {hasIllustration && <LayoutIllustration />}
       {hasMetabot ? <MetabotWidget /> : <HomeGreeting />}
       {isAdmin && (
-        <Tooltip tooltip="Pick a dashboard to serve as the homepage">
+        <Tooltip tooltip={t`Pick a dashboard to serve as the homepage`}>
```
Comment (WiNloSt) [inline frontend/src/metabase/home/components/HomeLayout/HomeLayout.tsx:35]: image

File: frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx [hunk 7]
```diff
@@ -45,7 +45,7 @@ export function DatabasePromptBanner({ location }: DatabasePromptBannerProps) {
             }}
           >
             <ConnectDatabaseButton small>
-              Connect your database
+              {t`Connect your database`}
```
Comment (WiNloSt) [inline frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx:48]: image

File: frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts [hunk 8]
```diff
@@ -1,16 +1,26 @@
...
+} from "metabase/lib/groups";
 
 const isPinnedGroup = (group: Group) =>
   isAdminGroup(group) || isDefaultGroup(group);
 
 export const getOrderedGroups = createSelector(
   Groups.selectors.getList,
-  (groups: Group[]) => _.partition(groups, isPinnedGroup),
+  (groups: Group[]) => {
+    const translatedGroups = groups.map(group =>
```
Comment (WiNloSt) [inline frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts:19]: Appears in
1. Admin > Permissions > Data > Group
    Screenshot 2023-08-09 at 6 14 29 PM

And a few other places that I'm too tired to track right now. But you can try to find the usage of `getOrderedGroups` to see which components this selector will be used on.

**Pull request merged**